buildlib/jucx/jucx-test.yml

```diff
@@ -0,0 +1,76 @@
+parameters:
+  name: subtest
+  demands: []
+
+jobs:
+  - job: jucx_${{ parameters.name }}
+    displayName: jucx ${{ parameters.name }}
+
+    pool:
+      name: MLNX
+      demands: ${{ parameters.demands }}
+
+    strategy:
+      matrix:
+        java8:
+          JAVA_VERSION: 1.8
+        java11:
+          JAVA_VERSION: 1.11
+
+    steps:
+      - checkout: self
+        clean: true
+        displayName: Checkout
+      - bash: |
+          set -eEx
+          source buildlib/az-helpers.sh
+          if module_load dev/jdk && module_load dev/mvn
+          then
+              ./autogen.sh
+              ./contrib/configure-devel --prefix=$(Build.Repository.LocalPath)/install --with-java --enable-gtest=no
+              make -j`nproc`
+              make install
+          else
+              azure_log_warning "No jdk and mvn modules"
+              exit 0;
+          fi
+        displayName: Build UCX
+      - bash: |
+          set -eEx
+          source buildlib/az-helpers.sh
+          if module_load dev/jdk && module_load dev/mvn
```

Comment:
if ! module load exit

Reply:
Please use dockers from harbor with setuped env like in my PR #6105

Reply:
Probably another PR? It's jucx tests that has to be run on agents.